avm/res/web/static-site/main.bicep

```diff
@@ -241,6 +241,7 @@ resource staticSite_roleAssignments 'Microsoft.Authorization/roleAssignments@202
 module staticSite_privateEndpoints 'br/public:avm/res/network/private-endpoint:0.4.0' = [
   for (privateEndpoint, index) in (privateEndpoints ?? []): {
     name: '${uniqueString(deployment().name, location)}-StaticSite-PrivateEndpoint-${index}'
+    scope: resourceGroup(privateEndpoint.?resourceGroupName ?? '')
     params: {
       name: privateEndpoint.?name ?? 'pep-${last(split(staticSite.id, '/'))}-${privateEndpoint.?service ?? 'staticSites'}-${index}'
       privateLinkServiceConnections: privateEndpoint.?manualPrivateLinkServiceConnections != true
```

Comment:
As per our conversation, please also update the `privateEndpoint.?manualPrivateLinkServiceConnections` assertions to `privateEndpoint.?isManualConnection`

Reply:
Done: https://github.com/Azure/bicep-registry-modules/pull/1472/commits/bafee36c2e2a9b9492f1a189d99672fe53822a70
Final test of all modules included in this PR in progress...